pkg/queue/timeout_test.go

```diff
@@ -76,9 +78,10 @@ func TestTimeToFirstByteTimeoutHandler(t *testing.T) {
 	}, {
 		name:    "custom timeout message",
 		timeout: failingTimeout,
-		handler: func(writeErrors chan error) http.Handler {
+		handler: func(mux *sync.Mutex, writeErrors chan error) http.Handler {
 			return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-				time.Sleep(sleepToFail)
+				mux.Lock()
```

Comment:
what's the difference from `timeout` case?

Reply:
Nothing but it configures a custom message to return.

Reply:
ah, it's the hidden part 🗡